.scripts/utils/chalk.ts

```diff
@@ -0,0 +1,12 @@
+import chalk from 'chalk';
+
+enum LogTypes {
+  success = 'green',
+  info = 'cyan',
+  error = 'red',
+  warning = 'yellow',
+}
+
+export function log(text: string, type: keyof typeof LogTypes) {
```

Comment:
I think this module should return the text with chalk, not do the console here. Let modules which use this do console.
```suggestion
export function makeUpText(text: string, type: keyof typeof LogTypes) {
   return chalk[LogTypes[type]](text);
```

Reply:
rename the file to "make-up-text.ts"

Reply:
The reason I suggest this one, because in the text, it could have some color texts, some don't